oclif.manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.7",
+  "version": "1.1.8",
```

Comment:
@vkalta Can you please do the version bump as 1.1.10 in package.json and package-lock files as the staging branch version is 1.1.9 which is about to be released in the next release